backend/src/test/java/com/redhat/cloud/notifications/routers/EventServiceTest.java

```diff
@@ -479,6 +479,26 @@ void testInvalidLimit() {
                 .contentType(JSON);
     }
 
+    @Test
+    void shouldNotBeAllowedToGetEventLogs() {
+        Header identityHeader = mockRbac(DEFAULT_ACCOUNT_ID, "noaccess", NO_ACCESS);
+        given()
+                .header(identityHeader)
+                .when().get(PATH)
+                .then()
+                .statusCode(403);
+    }
```

Comment:
This case is already tested here:
https://github.com/RedHatInsights/notifications-backend/blob/a103e4cd771007814295860501fc8d8fa0ed5d01/backend/src/test/java/com/redhat/cloud/notifications/routers/EventServiceTest.java#L441-L449

Reply:
oops :X